src/metadata.js

```diff
@@ -0,0 +1,125 @@
+const basePath = process.cwd();
+const buildDir = `${basePath}/build`;
+const { NETWORK } = require(`${basePath}/constants/network.js`);
+const { METADATA } = require(`${basePath}/constants/metadata.js`);
+const fs = require("fs");
+const sha1 = require(`${basePath}/node_modules/sha1`);
+const {
+  baseUri,
+  description,
+  extraMetadata,
+  namePrefix,
+  network,
+  solanaMetadata,
+  debugLogs,
+} = require(`${basePath}/src/config.js`);
+
+// create metadata item with the provided data
+const createMetadataItem = (attributesList, _dna, _edition) => {
+  let tempMetadata = {};
+
+  switch (network) {
+    case NETWORK.egld: {
+      tempMetadata = {
```

Comment:
If there's no extra logic after the `switch` statement, return the object directly